<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoDataPointmapRealTextImporter_SettingDialog</class>
 <widget class="QDialog" name="GeoDataPointmapRealTextImporter_SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
     <item>
      <widget class="QLabel" name="fileNameLabel">
       <property name="text">
        <string>File name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fileNameValueLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="encodingLabel">
           <property name="text">
            <string>Encoding:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="encodingComboBox"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="csLabel">
           <property name="text">
            <string>Coordinate System</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="CoordinateSystemConvertWidget" name="csWidget" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="formatGroupBox">
         <property name="title">
          <string>File Format</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="csvRadioButton">
            <property name="text">
             <string>CSV (Comma separated values)</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="customRadioButton">
            <property name="text">
             <string>Custom delimited text</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="delimitersGroupBox">
         <property name="title">
          <string>Delimiters</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="commaCheckBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Comma</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="tabCheckBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Tab</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="spaceCheckBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Space</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="colonCheckBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Colon</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="semicolonCheckBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Semicolon</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="otherCheckBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Other</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="otherLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0">
         <item>
          <widget class="QGroupBox" name="specialGroupBox">
           <property name="title">
            <string>Special characters</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0,0,0,0,1">
            <item>
             <widget class="QLabel" name="quoteLabel">
              <property name="text">
               <string>Quote character:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="quoteLineEdit">
              <property name="text">
               <string>&quot;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="TooltipLabel" name="quoteHelpLabel">
              <property name="toolTip">
               <string>When you want to specify multiple quote characters, please just input the characters only, no spaces or commas.</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../../guibase/guibase.qrc">:/libs/guibase/images/iconQuestion.svg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="escapeLabel">
              <property name="text">
               <string>Escape character:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="escapeLineEdit">
              <property name="text">
               <string>\</string>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="TooltipLabel" name="escapeHelpLabel">
              <property name="toolTip">
               <string>The character to escape quote characters in quoted text. You can specify only one character.</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../../guibase/guibase.qrc">:/libs/guibase/images/iconQuestion.svg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="recordOptionsGroupBox">
         <property name="title">
          <string>Record options</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="headerLinesLabel">
            <property name="text">
             <string>Header lines to ignore</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="headerLinesSpinBox"/>
          </item>
          <item>
           <widget class="QCheckBox" name="fieldNameCheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>First header line contains field names</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>199</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="selectFieldsGroupBox">
         <property name="title">
          <string>Select fields</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="xLabel">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="xFieldComboBox">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="yLabel">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="yFieldComboBox">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="valueLabel">
            <property name="text">
             <string>Value:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="valueFieldComboBox">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>73</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Skip setting</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="skipRateLabel">
            <property name="text">
             <string>Skip rate:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="skipRateSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TooltipLabel" name="skipRateHelpLabel">
            <property name="toolTip">
             <string>When you want to skip lines in the file, please specify values larger than 1. For example, if you input 2, every other line is imported.</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../../guibase/guibase.qrc">:/libs/guibase/images/iconQuestion.svg</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="errorsLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: red;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="previewGroupBox">
      <property name="title">
       <string>Preview</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CoordinateSystemConvertWidget</class>
   <extends>QWidget</extends>
   <header>cs/coordinatesystemconvertwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TooltipLabel</class>
   <extends>QLabel</extends>
   <header>guibase/tooltiplabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../guibase/guibase.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeoDataPointmapRealTextImporter_SettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeoDataPointmapRealTextImporter_SettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otherCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>otherLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
